This file extends `-smb' to add a share for the Nix store, and changes
the name of the default share.

--- a/net/slirp.c
+++ b/net/slirp.c
@@ -515,8 +515,12 @@ static int slirp_smb(SlirpState* s, const char *exported_dir,
             "log file=%s/log.smbd\n"
             "smb passwd file=%s/smbpasswd\n"
             "security = share\n"
-            "[qemu]\n"
-            "path=%s\n"
+            "[store]\n"
+            "path=/gnu/store\n"
+            "read only=yes\n"
+            "guest ok=yes\n"
+            "[xchg]\n"
+            "path=%s/xchg\n"
             "read only=no\n"
             "guest ok=yes\n",
             s->smb_dir,
